### ISSUE TITLE
Run clean-project-auto without React Native CLI

### ISSUE THREAD

**Issue by sonicdoe** [pmadruga/react-native-clean-project#45]
If `clean-project-auto` were available through the `react-native-clean-project` executable (or a separate `react-native-clean-project-auto` executable), [`npx`](https://github.com/npm/npx) could be used without having to install the React Native CLI.

In short, it would be useful to have `npx react-native-clean-project --auto`.

**Comment by pmadruga** [comment]
What happens when you run `npx react-native-clean-project`?

**Comment by sonicdoe** [comment]
It runs [the main executable](https://github.com/pmadruga/react-native-clean-project/blob/fd53ac8b4666959d084f613a7f4fffb95d116efb/package.json#L9). Therefore, it asks “Wipe iOS build folder? (Y/n)”.

**Comment by plaa** [comment]
I was trying to find a command line argument for this as well. Also `npx react-native-clean-project --help` just starts the interactive wiping.

Command line help should be provided with `-h`/`--help` and unknown command line parameters should cause an error (like every other command line tool).

**Comment by gwendall** [comment]
+1

**Comment by pmadruga** [comment]
Just to be sure, what you're requesting it the ability to do the exact same as `clean-project-auto` (meaning, that there's no `brew update/upgrade`, no `yarn install` and no `pod update` ), right?

Shouldn't be a problem, going to aim to work on it this week or the next.

**Comment by sonicdoe** [comment]
Exactly. In other words, provide `react-native clean-project-auto` through [direct execution](https://github.com/pmadruga/react-native-clean-project#direct-execution).

**Comment by pmadruga** [comment]
What this would create is some opinionated cleanup. Meaning that it would run all the commands. Is this OK for everyone?

**Comment by sonicdoe** [comment]
Do you mean _all_ commands that `react-native clean-project-auto` would run (for example, _not_ `pod update`)?

**Comment by pmadruga** [comment]
Yes, if that's what the community wants. What do you suggest?

**Comment by sonicdoe** [comment]
I think that’s good 👍 After all, that’s exactly how the existing `react-native clean-project-auto` command already works.
